<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMain</class>
 <widget class="QDialog" name="DialogMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>344</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layContent">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QListWidget" name="list">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Themes</string>
        </property>
        <property name="icon">
         <iconset theme="applications-graphics"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Font</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-desktop-font"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sound</string>
        </property>
        <property name="icon">
         <iconset theme="audio-volume-high"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-desktop"/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page1">
        <layout class="QGridLayout" name="layThemes">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="lblColorScheme">
           <property name="text">
            <string>Color scheme:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblGtkTheme">
           <property name="text">
            <string>GTK theme:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cbxIconThemeName"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cbxCursorThemeName"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbxThemeName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblCursorThemeName">
           <property name="text">
            <string>Cursor theme:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblIconThemeName">
           <property name="text">
            <string>Icon theme:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbxColorScheme"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblCursorThemeSize">
           <property name="text">
            <string>Cursor size:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QWidget" name="wgtPreview" native="true"/>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="sbxCursorThemeSize">
           <property name="maximum">
            <number>512</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chkPreferDarkTheme">
           <property name="text">
            <string>Prefer dark theme</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page2">
        <layout class="QGridLayout" name="layFont">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkXftAntialias">
           <property name="text">
            <string>Enable Antialiasing</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkXftHinting">
           <property name="text">
            <string>Enable Hinting</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QComboBox" name="cbxXftRgba"/>
         </item>
         <item row="12" column="0" colspan="3">
          <spacer name="vspFont">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblXftDpi">
           <property name="text">
            <string>Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lblDefaultFont">
           <property name="text">
            <string>Default:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblXftHintStyle">
           <property name="text">
            <string>Hinting style:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="cbxXftHintStyle"/>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="FontButton" name="fbnDefaultFont">
           <property name="text">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QSpinBox" name="sbxXftDpi">
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblXftRgba">
           <property name="text">
            <string>RGBA order:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="cbxFontAntialiasing"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblFontAntialiasing">
           <property name="text">
            <string>Antialiasing:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="chkEnableInputFeedbackSounds">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Play event sounds as feedback to user input</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="chkAllowVolumeAbove100">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Allow volume above 100%</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="chkEnableEventSounds">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Play event sounds</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblSoundThemeName">
           <property name="text">
            <string>Sound theme:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <spacer name="vspSound">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbxSoundThemeName"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkOverlayScrolling">
           <property name="text">
            <string>Enable overlay scrolling</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkWarpSlider">
           <property name="text">
            <string>Primary button warps slider</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbxScaleFactor">
           <property name="title">
            <string notr="true">Scale factor</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string notr="true">General:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string notr="true">Text:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sbxScaleFactor">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="dsbTextScaleFactor">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbxEnableRecentFiles">
           <property name="title">
            <string>Recent files</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="lblRecentFilesMaxAge">
              <property name="text">
               <string>Max age:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sbxRecentFilesMaxAge">
              <property name="minimum">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="vspOther">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="LayButtons">
     <item>
      <widget class="QPushButton" name="pbnAbout">
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset theme="help-about"/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FontButton</class>
   <extends>QPushButton</extends>
   <header>src/FontButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
